Userland/Services/FileSystemAccessServer/ClientConnection.h

```diff
@@ -26,11 +26,13 @@ class ClientConnection final
     virtual void die() override;
 
 private:
+    virtual void request_file_read_only_approved(i32, i32, String const&) override;
     virtual void request_file(i32, i32, String const&, Core::OpenMode const&) override;
     virtual void prompt_open_file(i32, i32, String const&, String const&, Core::OpenMode const&) override;
     virtual void prompt_save_file(i32, i32, String const&, String const&, String const&, Core::OpenMode const&) override;
 
     void prompt_helper(Optional<String> const&, Core::OpenMode const&);
+    void request_file_handler(i32, i32, String const&, Core::OpenMode const&, bool);
```

Comment:
These poor parameters could use some names...or better types, in case of that bool

Reply:
The rest of the parameters except the last `bool` are the same as the ones for `request_file`, so I left them the same. 

What type should I use instead of a bool? The value is supposed to be either true / false (whether to prompt the user or not). I'll add a name for it but I'm not sure what else to do?

Reply:
In serenity we generally don't use bools for functions that receive a constant at the call site, since they're usually not very verbose, eg:
```c++
some_func(true, false);
```
Instead we create a Yes/No enum class, which results in much more verbose call sites, eg:
```c++
some_func(DoThing::Yes, SomeOtherThing::No);
```
So in your case you might add the following:
```c++
enum class ShouldPrompt {
No,
Yes
}
```

Reply:
(check https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#wrong-7 for more info)

Reply:
@IdanHo That's very helpful, thanks! I will make this change.